java/arcs/android/storage/service/IRegistrationCallback.aidl

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.android.storage.service;
+
+/** Mechanism allowing an asynchronous response from the StorageService. */
+interface IRegistrationCallback {
+    oneway void onSuccess(in int token);
+
+    oneway void onFailure(in byte[] exception);
```

Comment:
Some documentation comments here explaining what the `byte[]` represents would be good. (e.g. that they're a protobuf serialization of a CrdtException).

Reply:
will do doc/cleanup pass